src/main/java/rx/internal/operators/OperatorTakeLastOne.java

```diff
@@ -0,0 +1,162 @@
+package rx.internal.operators;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import rx.Observable.Operator;
+import rx.Producer;
+import rx.Subscriber;
+
+public class OperatorTakeLastOne<T> implements Operator<T, T> {
+
+    private static class Holder {
+        static final OperatorTakeLastOne<Object> INSTANCE = new OperatorTakeLastOne<Object>();
+    }
+
+    @SuppressWarnings("unchecked")
+    public static <T> OperatorTakeLastOne<T> instance() {
+        return (OperatorTakeLastOne<T>) Holder.INSTANCE;
+    }
+
+    private OperatorTakeLastOne() {
+
+    }
+
+    @Override
+    public Subscriber<? super T> call(Subscriber<? super T> child) {
+        final ParentSubscriber<T> parent = new ParentSubscriber<T>(child);
+        child.setProducer(new Producer() {
+
+            @Override
+            public void request(long n) {
+                parent.requestMore(n);
```

Comment:
I think parent may request Long.MAX_VALUE because it will only hold onto the very last.